frontend/public/locales/en/translation.json

```diff
@@ -1440,6 +1441,7 @@
   "hostCount_plural": "{{count}} hosts",
   "Hosted": "Hosted",
   "Hosted cluster requires a manual import.": "Hosted cluster requires a manual import.",
+  "Hosted clusters cannot be destroyed.": "Hosted clusters cannot be destroyed.",
```

Comment:
I think this is kind of misleading since users can still destroy the hosted cluster from the CLI. Maybe we can change this to say:
`Hosted clusters cannot be destroyed from the console. Use the individual cluster destroy option to see CLI instructions.`

Reply:
Thanks @fxiang1 , yeah that is much more user friendly. Will add that.